### PR TITLE
Paddle moving

### LINKED ISSUE #4: Paddle Moving
As a player,
So that I can play Ping,
I would like to be able to move the paddle up and down.

### PR DESCRIPTION
**Pull request by Hempy49**
closes #4

### CODE REVIEW

File: src/paddle.js [hunk 1]
```diff
@@ -1,21 +1,40 @@
...
+Paddle.prototype.moveUp = function () {
+  if (this.location > 0) {
+    this.location -= this.SPEED;
+  } else {
+    this.location = 0;
+    }
+  };
+
+Paddle.prototype.moveDown = function () {
+  if (this.location + this.height < this.canvas.height) {
```
Comment (alistairkung) [inline src/paddle.js]: Can we have a private method for (this.location + this.height < this.canvas.height)? e.g. topLimit/bottomLimit

**Review by alistairkung — CHANGES_REQUESTED** [review]
one change

**Review by alistairkung — APPROVED** [review]

**Comment by alistairkung** [comment]
🐴

**Review by terminalobject — APPROVED** [review]
🦑

**Pull request merged**